plugins/invisible-typing/src/Icon.tsx

```diff
@@ -17,43 +17,41 @@ export const Icon = (props: { type?: { analyticsName: string } }) => {
   const channelWise = cfg.get("button", true) ? cfg.get("channelWise", true) : false;
   const channelsList = cfg.get("channels", { [channelId]: globalInvisible });
   const [enabled, setEnabled] = React.useState(
-    channelWise ? channelsList[channelId] : globalInvisible,
+    channelWise ? channelsList[channelId] ?? globalInvisible : globalInvisible,
   );
   return (
-    <>
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (performance):** Removal of fragment in favor of div with key improves re-render optimization.

```suggestion
<div key={enabled ? 'enabled' : 'disabled'}>
```